contracts/libs/gov/gov-pool/GovPoolCreate.sol

```diff
@@ -154,20 +168,45 @@ library GovPoolCreate {
         snapshotId = IGovUserKeeper(userKeeper).createNftPowerSnapshot();
     }
 
-    function _restrictInterestedUsersFromProposal(
+    function _banTreasury(
         mapping(address => IGovPool.UserInfo) storage userInfos,
         IGovPool.ProposalAction[] calldata actions,
         uint256 proposalId
-    ) internal {
+    ) internal returns (uint256 bannedTreasury) {
+        IGovPool govPool = IGovPool(address(this));
+
+        (, address userKeeperAddress, , , ) = govPool.getHelperContracts();
+        (, address expertNft, address dexeExpertNft, ) = govPool.getNftContracts();
+
+        IGovUserKeeper userKeeper = IGovUserKeeper(userKeeperAddress);
+
         for (uint256 i; i < actions.length; i++) {
             IGovPool.ProposalAction calldata action = actions[i];
+            bytes4 selector = action.data.getSelector();
```

Comment:
Will this work with empty calldata? Like when ether is transferred

Reply:
It'll return bytes4(0) selector